### ISSUE TITLE
Allow point selection on non-time plots

### ISSUE THREAD

**Issue by squaregoldfish** [BjerknesClimateDataCentre/QuinCe#431]
Currently we can only click points (to highlight them in the table) on plots where time is the X axis. This should be available on all plots.

Depends on #405 being done first.

**Comment by squaregoldfish** [comment]
Duplicate of #393
